src/vs/workbench/contrib/chat/browser/chatWidget.ts

```diff
@@ -167,6 +168,27 @@ export class ChatWidget extends Disposable implements IChatWidget {
 		this.inputPart.focus();
 	}
 
+	focusWithId(id: string, type: 'next' | 'previous'): void {
+		const items = this.viewModel?.getItems();
+		if (!items) {
+			return;
+		}
+		const focusedElement = items.find(i => i.id === id);
```

Comment:
elementToFocus?

Reply:
no, this is the currently focused element.